mantidimaging/gui/dialogs/async_task/view.py

```diff
@@ -63,6 +69,10 @@ def set_progress(self, progress: float, message: str):
         # Update progress bar
         self.progressBar.setValue(int(progress * 1000))
 
+    def set_progress_plot(self, x, y):
```

Comment:
Type annotations

Reply:
Thanks. Had meant to add these and completely forgot